src/components/Courses/Components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,21 @@
+import { Button } from '../../../../common/Button/Button';
+
+type SearchBarProps = {
+  children: React.ReactNode;
+};
+
+export const SearchBar = (props: SearchBarProps) => {
+  return (
+    <div>
```

Comment:
Vertical alignment between the input field and the button is off.

![image](https://github.com/elenapetra/react-app/assets/145835962/8e07e973-73f0-4ec2-8101-3e36e4437017)

Reply:
Fixed